README.md

```diff
@@ -25,8 +25,17 @@ In this challenge use `Test Driven Development` to build a RESTful API using Nod
 Demonstrate your understanding of this week's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
 1. In Jest, what are the differences between `describe()` and `it()` globals, and what are good uses for them?
-1. What is the point of `Test Driven Development`? What do you think about this approach?
-1. Mention three types of automated tests.
+
+    `describe()` is the container that multiple `it()`'s can go into, where the `it()`'s are the test performed
+
+2. What is the point of `Test Driven Development`? What do you think about this approach?
+
+    you are making the app tests before the app itself, it cant get any more tested than that LOL
```

Comment:
What is the point of testing the application and setting up tests before developing? What are the benefits of test-driven development?